### DISCUSSION TITLE
CVEs found in most recent build

### DISCUSSION THREAD

**Discussion by sbirl** [prymitive/karma#5868] [General]
Our security scanner picked up the following CVEs when using your latest image (0.116):

* CVE-2023-4911, **HIGH**,  affecting libc6 `2.31-13+deb11u6`.  Fixed in `2.31-13+deb11u7`.
* CVE-2023-3446, MEDIUM, affecting libssl1.1 `1.1.1n-0+deb11u5`.  Fixed in `1.1.1v-0~deb11u1`.
* CVE-2023-3817, MEDIUM, affecting openssl `1.1.1n-0+deb11u5`.  Fixed in `1.1.1v-0~deb11u1`.

**Comment by sbirl** [comment]
New CVEs have been identified since my initial post on October 9th:

- CVE-2023-4806, Medium
- CVE-2023-4813, Medium
- CVE-2010-4756, Low
- CVE-2018-20796, Low
- CVE-2019-1010022, Low
- CVE-2019-1010023, Low
- CVE-2019-1010024, Low
- CVE-2019-1010025, Low
- CVE-2019-9192, Low
- CVE-2007-6755, Low
- CVE-2010-0928, Low

**Comment by sfudeus** [comment]
According to trivy, 0.116 had 20 CVEs (based on debian 11.7). 
0.117+ is based on debian 11.8 and has none, according to trivy:

```
lmierzwa/karma:v0.116 (debian 11.7)

Total: 20 (UNKNOWN: 0, LOW: 11, MEDIUM: 8, HIGH: 1, CRITICAL: 0)
```

```
lmierzwa/karma:v0.117 (debian 11.8)

Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

```
lmierzwa/karma:v0.118 (debian 11.8)

Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```
